src/main/java/org/eaa690/aerie/communication/TwilioSMSSender.java

```diff
@@ -0,0 +1,32 @@
+package org.eaa690.aerie.communication;
+
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+
+import org.eaa690.aerie.constant.PropertyKeyConstants;
+import org.eaa690.aerie.exception.ResourceNotFoundException;
+import org.eaa690.aerie.service.PropertyService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TwilioSMSSender implements MessageSender {
```

Comment:
This is effectively the same as the existing SMSService. Simply modify the SMSService with the MessageSender interface and modify the SMSService class to adhere to the interface